src/memline.c

```diff
@@ -4236,6 +4237,7 @@ ml_flush_line(buf_T *buf)
 	    }
 	}
 	vim_free(new_line);
+	buf->b_ml.ml_line_size = 0;
```

Comment:
don't you ned this a few lines below, when freeing `buf->b_ml.ml_line_ptr`?

Reply:
I don't think it is necessary, but it should be safer.
Also clearing it in textprop.c might be better.